science/cdo/Portfile

```diff
@@ -4,19 +4,19 @@ PortSystem                  1.0
 PortGroup                   mpi 1.0
 
 name                        cdo
-version                     2.0.5
+version                     2.0.6
 revision                    1
```

Comment:
the `revision` field should be reset to `0` when updating the version

Reply:
Oops, forgot. Thanks for spotting that. Just fixed it and squashed the commits.